streams/shared/src/main/scala/zio/stream/ZChannel.scala

```diff
@@ -1633,23 +1633,27 @@ object ZChannel {
     private[zio] final case class Finalizer[Env, OutErr, OutDone](finalizer: Exit[OutErr, OutDone] => URIO[Env, Any])
         extends Continuation[Env, Any, Any, Any, OutErr, Nothing, Nothing, OutDone, Nothing]
 
+    @deprecated("use successIdentity", "2.1.18")
```

Comment:
This should be safe to remove since it's `private[this]`. I meant to deprecate the method below that is `private[zio]`